content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: CEPH CSI User Guide description: "A user guide for integrating SODA with CEPH CSI Driver via unified Plugin and
+hotpot project"
```

Comment:
Please fix the frontmatter. Please fix the newlines.

Reply:
@anvithks I have done the changes, please check.